sdk-java/src/main/java/io/littlehorse/sdk/common/config/LHConfig.java

```diff
@@ -268,6 +272,27 @@ private Channel getChannel(String host, int port) throws IOException {
         return out;
     }
 
+    /**
+     * Get a blocking stub with the application defaults
+     */
+    private LittleHorseBlockingStub getDefaultBlockingStub(String host, int port) throws IOException {
+        String tenantId = getTenantId();
+        LittleHorseBlockingStub blockingStub = LittleHorseGrpc.newBlockingStub(getChannel(host, port));
+        if (tenantId != null) {
+            return blockingStub.withCallCredentials(new TenantMetadataProvider(tenantId));
+        }
+        return blockingStub;
+    }
+
+    private LittleHorseStub getDefaultAsyncStub(String host, int port) throws IOException {
```

Comment:
What does "default" mean here?

Reply:
The application default setup to communicate with the server. method clients can add more configuration on top of this default configurations. For example, this method can return the default setup and on top of this we can add the OAuth configuration.